apps/studio/prisma/generated/generatedEnums.ts

```diff
@@ -7,6 +7,7 @@ export const ResourceType = {
   RootPage: "RootPage",
   Page: "Page",
   Folder: "Folder",
+  Blog: "Blog",
```

Comment:
Why is this a new resource type btw? I would have imagined it to be a new layout actually

Reply:
sadly, `layout` is a **blob only** concept. for folders, what we actually do is to generate the layout ourselves as part of publishing.

you can verify this yourself by tracing the flow in `publishing/index` -> `processDanglingDirectory` -> `getFolderIndexPageContents` (for example), where we write the layout there ourselves.

Reply:
because this is (as rightly mentioned) a layout (in the sense that it dictates how the cards etc layout), we can't actually do this -> we'd have to pass it as `ResourceTyp`

Reply:
because the main difference is how we display the cards, would't a new key like "format: blog" solve this?

Reply:
yes and no - we'd need _something_ to tell the component how to display, but adding a new key like this would require adding new column in our db too, and involve quite abit of extra work in publishing (to persist to disk) and components (to know that they'd need to read this new key and render conditionally based on this key)